config/routes.rb

```diff
@@ -1,11 +1,12 @@
 IntergalacticBattleship::Application.routes.draw do
+  devise_for :users
   root "strikes#index"
   resources :strikes, only: [:new, :create]
   # The priority is based upon order of creation: first created -> highest priority.
   # See how all your routes lay out with "rake routes".
 
   # You can have the root of your site routed with "root"
-  root 'static_pages#index'
```

Comment:
So does this mean delete the line about static_pages#index and it's controller file?